src/Functions/toMillisecond.cpp

```diff
@@ -0,0 +1,20 @@
+#include <Functions/FunctionFactory.h>
+#include <Functions/DateTimeTransforms.h>
+#include <Functions/FunctionDateOrDateTimeToSomething.h>
+#include <DataTypes/DataTypesNumber.h>
+
+
+namespace DB
+{
+
+using FunctionToMillisecond = FunctionDateOrDateTimeToSomething<DataTypeUInt8, ToMillisecondImpl>;
+
+REGISTER_FUNCTION(ToMillisecond)
+{
+    factory.registerFunction<FunctionToMillisecond>();
+
+    /// MySQL compatibility alias.
+    factory.registerAlias("MILLISECOND", "toMllisecond", FunctionFactory::CaseInsensitive);
```

Comment:
`toMllisecond`
may be `toMillisecond`?

Reply:
Some test is required.

Reply:
> `toMllisecond` may be `toMillisecond`?

oh my bad sorry i'll just fix that real quick

Reply:
You could just update it here. No need to close this PR.

Reply:
oh, I didn't realize I could do that, thanks. Btw I made another PR it would be great if you could review that as I had some doubts too